components/browser/toolbar/src/main/res/drawable/background_with_ripple.xml

```diff
@@ -0,0 +1,7 @@
+<!-- This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
Was this meant to be in browser-toolbar or in sample browser? If we're keeping it here, we should rename it to follow our namespacing to avoid naming collisions: `mozac_browser_toolbar_background_with_ripple`

Reply:
> Was this meant to be in browser-toolbar or in sample browser? 

Sample browser, though I don't see a drawables folder for it? Should I just make one? @jonalmeida

Reply:
Yeah!